solutions/swb-reference/integration-tests/config/release-candidate-private.yaml

```diff
@@ -21,6 +21,9 @@ projectAdmin2PasswordParamStorePath: '/swb/release-candidate-private/pa2/passwor
 researcher1UserNameParamStorePath: '/swb/release-candidate-private/researcher1/email'
 researcher1PasswordParamStorePath: '/swb/release-candidate-private/researcher1/password'
 
+## Default external ID
+externalId: 'workbench'
```

Comment:
Can we have this stored in an SSM parameter instead? This will prevent us storing the value publicly